src/api/protectedApiClient.ts

```diff
@@ -24,6 +28,7 @@ export enum ProtectedApiClientRoutes {
   REQUEST_STATUSES = '/api/v1/protected/requests/status',
   MAKE_PF_REQUEST = 'api/v1/protected/requests',
   USER = '/api/v1/protected/user',
+  ANNOUNCEMENTS = 'api/v1/protected/announcements',
```

Comment:
❓ Is the announcements route supposed to be protected? I thought the single event page was accessible to everyone, so I just want to confirm that only logged in users can see updates for an event